### ISSUE TITLE
If there a way to prevent searching when querystring is empty?

### ISSUE THREAD

**Issue by mingfang** [YousefED/ElasticUI#54]
This is my search input
```
  <input placeholder="Search" type="text" eui-query="ejs.MatchQuery('line', querystring)" ng-model="querystring" eui-enabled="querystring.length" />
```
This work during startup since eui-enabled will be false.
However, after doing a search and then subsequently clearing the search input, the result is another search using an empty query.

My workaround is to surround the result display with this
```
    <div ng-if="indexVM.query">
      <ul>
          <li ng-repeat="doc in indexVM.results.hits.hits">
...
          </li>
      </ul>
    </div>
```
Ideally I want to avoid the search completely.

**Comment by YousefED** [comment]
I don't think this is currently supported. You might be able to unset (null / empty) indexVM.index to accomplish what you want, but this would also be kind of a workaround

**Comment by mingfang** [comment]
#40 seems to be a solution but that PR has been stuck there since May.
